pilot/pkg/serviceregistry/serviceentry/servicediscovery_test.go

```diff
@@ -519,6 +521,8 @@ func TestServiceDiscoveryWorkloadUpdate(t *testing.T) {
 	t.Run("add workload", func(t *testing.T) {
 		// Add a WLE, we expect this to update
 		createConfigs([]*config.Config{wle}, store, t)
+		//expectEvents(t, events, Event{kind: "xds", proxyIP: "2.2.2.2"})
```

Comment:
You can remove this if it is not required anymore.

Reply:
Thanks